modes/segmentation/src/index.tsx

```diff
@@ -118,18 +122,42 @@ function modeFactory({ modeConfiguration }) {
         'SegmentationUtilities',
         'SegmentationTools',
       ]);
+      toolbarService.updateSection(toolbarService.sections.labelMapSegmentationToolbox, [
+        'LabelMapUtilities',
+        'LabelMapTools',
+      ]);
+      toolbarService.updateSection(toolbarService.sections.contourSegmentationToolbox, [
+        'ContourUtilities',
+        'ContourTools',
+      ]);
+
       toolbarService.updateSection('SegmentationUtilities', [
         'LabelmapSlicePropagation',
         'InterpolateLabelmap',
         'SegmentBidirectional',
         'SegmentLabelTool',
       ]);
+      toolbarService.updateSection('LabelMapUtilities', [
```

Comment:
please add a comment here about what we have in mind here in long run, since one might ask why we have SegmentationUtilities and Labelmaputils and contour utils etc etc